### DISCUSSION TITLE
@updatedAt updates unneccessarily

### DISCUSSION THREAD

**Discussion by Aniket-Jena-02** [prisma/prisma#14594] [Q&A] [answered]
I have the following schema

```prisma
id        String   @id @default(uuid())
createdAt DateTime @default(now())
updatedAt DateTime @updatedAt
title     String   @db.VarChar(255)
content   String
votes     Int      @default(0)
author    User     @relation(fields: [authorId], references: [id])
updated   Boolean  @default(false)
voted     Boolean  @default(false)
authorId  String
```
Whenever votes change updated at also changes, how can I prevent this to make updatedAt record time only when title on content change?

**Comment by austincrim** [comment, marked as answer]
Hey @killerdroid99 👋,

`@updatedAt` will always update whenever the row is changed. It is not possible to only watch specific fields for changing the `@updatedAt` field. 

You could create a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if this is something you would like to see added to Prisma.

**Reply by Aniket-Jena-02** [reply]
thanks, for now I totally separated any vote related columns to a separate table and established many to many relationship with users and posts 😅

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
